Gemfile.lock

```diff
@@ -224,6 +224,3 @@ DEPENDENCIES
   therubyracer
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
-
-BUNDLED WITH
-   1.10.4
```

Comment:
This shouldn't be removed from this file. I would remove the changes from this PR.